contracts/options.cairo

```diff
@@ -185,6 +188,27 @@ func mint_option_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_che
                 strike_price=strike_price,
             );
         }
+
+        with_attr error_message("trade exceeds max liqpool deposit size"){
```

Comment:
`trade exceeds max liqpool deposit size` shouldn't be something like `trade exceeds max option size`... or something like this?